docs/kratos/session-management/01_overview.mdx

```diff
@@ -150,36 +122,53 @@ This flow is similar to [GitHub's sudo mode](https://help.github.com/en/github/a
 The session is considered privileged when its `authenticated_at` isn't older than the `privileged_session_max_age` value defined
```

Comment:
```suggestion
The session is considered privileged when its `authenticated_at` is younger than the `privileged_session_max_age` value defined
```
I don't know, rephrase this maybe? reading "isn't older than" was tough to understand. Had to read it a few times to get the grasp of it. 

Does it mean that the session privileged allows editing or disallows editing / changing username / email.. etc.?

Reply:
Yeah, privileged sessions allow/enable editing fields like email